tox.ini

```diff
@@ -9,14 +9,12 @@ deps = pytest
        pytest-mock
 
 [testenv:py27]
-deps =
-    {[testenv]deps}
-    mock
+deps = {[testenv]deps}
+       mock
 
 [testenv:pypy]
-deps =
-    {[testenv]deps}
-    mock
+deps = {[testenv]deps}
+       mock
```

Comment:
I agree with @michaeljoseph . Is this mock needed here as well?

Otherwise LGTM